src/main/java/teammates/storage/sqlentity/responses/FeedbackConstantSumResponse.java

```diff
@@ -41,6 +41,6 @@ public String toString() {
      */
     @Converter
     public static class FeedbackConstantSumResponseDetailsConverter
-            extends JsonConverter<FeedbackConstantSumResponseDetails> {
+            extends FeedbackQuestionDetailsConverter {
```

Comment:
if i'm not wrong, these are response details, and not question details and hence the converter here is incorrect (need to create a `FeedbackResponseDetailsConverter`)

Reply:
Yes! Thanks for the catch!